cf/latest-snapshot/src/index.ts

```diff
@@ -7,31 +9,56 @@ function basename(path: String) {
 }
 
 // Directly fetch the data for the latest snapshot of a given chain (eg. calibnet or mainnet)
-async function get_latest(env: Env, chain: string): Promise<Response> {
+async function get_latest(req_headers: Headers, env: Env, chain: string): Promise<Response> {
 	const listed = await env.FOREST_ARCHIVE.list({ prefix: chain + "/latest/" });
 	let latest = listed.objects.at(-1);
 	if (latest == null) {
 		return new Response(`No latest snapshot found ${chain}`, {
 			status: 404,
 		});
 	} else {
-		// Should we support range queries?
-		const object = await env.FOREST_ARCHIVE.get(latest.key);
+		const object = await env.FOREST_ARCHIVE.get(latest.key, {
+			range: req_headers,
```

Comment:
Interesting. I didn't know you could pass in the headers directly for the `range` field. The documentation indicates that you cannot.

Reply:
What documentation are you referring to? The API seems to be happy with it.
```
  get(
    key: string,
    options: R2GetOptions & {
      onlyIf: R2Conditional | Headers;
    }
  ): Promise<R2ObjectBody | R2Object | null>;
```
...
```
declare interface R2GetOptions {
  onlyIf?: R2Conditional | Headers;
  range?: R2Range | Headers;
}
```